pt_miniscreen/__main__.py

```diff
@@ -24,7 +25,62 @@
 @click_logging.simple_verbosity_option(logger)
 @click.version_option()
 def main() -> None:
-    app = App()
+    from pitop import Pitop
+
+    from .tile_group import TileGroup
+    from .tile_group_context import TileGroupContext, TileGroupContextManager
+    from .tiles import HUDMenuTile, SettingsMenuTile, SettingsTitleBarTile
+    from .tiles.settings_connection import SettingsConnectionMenuTile
+
+    logger.debug("Setting ENV VAR to use miniscreen as system...")
+    environ["PT_MINISCREEN_SYSTEM"] = "1"
+
+    miniscreen = Pitop().miniscreen
+
+    title_bar_height = 19
+
+    main = TileGroupContext(
+        tile_groups=[
+            TileGroup(
+                size=miniscreen.size,
+                menu_tile=HUDMenuTile(size=miniscreen.size),
+                title_bar_tile=None,
+            ),
+            TileGroup(
+                size=miniscreen.size,
+                title_bar_tile=SettingsTitleBarTile(
+                    size=(miniscreen.size[0], title_bar_height),
+                    pos=(0, 0),
+                ),
+                menu_tile=SettingsMenuTile(
+                    size=(
+                        miniscreen.size[0],
+                        miniscreen.size[1] - title_bar_height,
+                    ),
+                    pos=(0, title_bar_height),
+                ),
+            ),
+        ],
+    )
+    settings_connection = TileGroupContext(
+        name="settings_menu",
+        tile_groups=[
+            TileGroup(
+                size=miniscreen.size,
+                menu_tile=SettingsConnectionMenuTile(miniscreen.size),
+                title_bar_tile=SettingsTitleBarTile(
+                    size=(
+                        miniscreen.size[0],
+                        miniscreen.size[1] - title_bar_height,
+                    ),
+                    pos=(0, title_bar_height),
+                    text="Settings / Connection",
```

Comment:
I would say that we need to have the settings connection context inherit from `TileGroupContext` and have behaviour extended to include handling of the title bar's text. Change of menu then emits events that the settings connection tile group context registers for and updates text accordingly.